### ISSUE TITLE
Navbar at the top covers game wrapper

### ISSUE THREAD

**Issue by timkchan** [colinschoen/Ants-Web-Viewer#17]
![screen shot 2016-07-06 at 16 44 05](https://cloud.githubusercontent.com/assets/19479507/16638066/f7e4853e-4398-11e6-9fc2-a2c5690e64bd.png)
adding some margin to fix the problem, please.
![screen shot 2016-07-06 at 16 44 21](https://cloud.githubusercontent.com/assets/19479507/16638065/f7dde45e-4398-11e6-952c-525fbbe8c7b4.png)

**Comment by colinschoen** [comment]
Try restarting the GUI and refreshing. It looks like something wasn't properly rendered. I am not showing this with the current version.

**Comment by timkchan** [comment]
tried refreshing which made the navbar goes away. is this the expected behaviour?
![screen shot 2016-07-06 at 17 08 09](https://cloud.githubusercontent.com/assets/19479507/16638451/43303454-439c-11e6-9065-3d75daae619f.png)

**Comment by colinschoen** [comment]
This is the expected UI:

<img width="831" alt="screen shot 2016-07-06 at 5 08 45 pm" src="https://cloud.githubusercontent.com/assets/2927722/16638467/57746192-439c-11e6-82ad-1081e7ba6da4.png">

Hmm, you are on Chrome?

**Comment by timkchan** [comment]
yes, latest chrome, mac

**Comment by colinschoen** [comment]
If you open `gui.py` what is `VERSION` _(line 14)_?

**Comment by timkchan** [comment]
v1.1
but doesn't it get update?
def update():
    request = urllib.request.Request("https://api.github.com/repos/colinschoen/Ants-Web-Viewer/releases/latest")
...

**Comment by colinschoen** [comment]
Yes, assuming everything goes well with the updater and your internet. 

I'll take a closer look tonight/tomorrow. 

Also are you CS61A summer staff or an ahead student?

**Comment by timkchan** [comment]
actually i took the class back in summer 15, I m trying to upload all the projects to my personal site. I m refactoring and testing the codes to make sure they work. I mean if u re busy, don't worry too much about it. I think u are my 61A TA actually.

**Comment by colinschoen** [comment]
Ah ok, excellent. 

No, it needs to be sorted out if it is an issue because people will be using it in a week or so. I'll keep you updated. Thanks for opening the issue. :)

**Comment by timkchan** [comment]
ya, np. I will try to look it to it more tonight. Thx!

**Comment by timkchan** [comment]
Collin, I just restarted chrome and it's working now. I suspect it's just chrome misbehaving. Thanks for your help!

**Comment by timkchan** [comment]
wait, it happens again. my bad. this is weird.

**Comment by timkchan** [comment]
ok, I think I figured out the problem. So, the project that I have is v1.0, but the update doesn't happen until the end of the game. So once the game ends, it runs the update, so when I ran the game again, now it will be in v1.1 and that fixed the problem. I will try to test more.

![screen shot 2016-07-06 at 17 35 09](https://cloud.githubusercontent.com/assets/19479507/16638926/03793c6c-43a0-11e6-9a38-b3b155a79451.png)

**Comment by colinschoen** [comment]
Ok, perhaps I can add a flag to force updates prior to running.

**Comment by timkchan** [comment]
Thx Colin. Also, I made a README.md for Ants. You are welcome to use and modify it if you would like to.
https://github.com/timkchan/ants
Thanks for all your help.

**Comment by colinschoen** [comment]
That's great, thanks. Would you be able to make a PR or are you ok with me just ripping it and committing it.

Also please don't post the project solution code in a public repo. Posting the GUI and skeleton project is fine, but the solution code should stay private.

**Comment by timkchan** [comment]
How similar is the current project compare to last summer? I put it here because recruiters always ask to see the code of projects. If it's very similar, i can remove/encrypt the ant.py file. However, student could go back to previous commit to see it unless i use bfg.

You can just rip it. If u can add something like 'written/contributed by a former 61A student Tim K. Chan' at the bottom, it'd be appreciated. Beware that i resized all the icon to 80x80 hence the folder icon8080, u might want to rip that one as well and update the links in the table.

**Comment by colinschoen** [comment]
It is very similiar. Most students aren't devious enough to look back in the history of a repo and just will use the github search feature to look for past code. If you have the time to use BFG that would be excellent, but please at least make sure it is removed from the head of the repo. 

I'll add the readme and attribution.
